packages/node-integration-tests/suites/tracing-new/auto-instrument/mysql/withoutConnect/test.ts

```diff
@@ -14,6 +14,9 @@ test('should auto-instrument `mysql` package.', async () => {
         op: 'db',
         data: {
           'db.system': 'mysql',
+          'server.address': 'localhost',
+          'server.port': 3306,
+          'db.user': 'root',
```

Comment:
just to sanity-check (becaus afaict `assertSentryTransaction` doesn't require strict equality): We don't collect something like `db.password`, rigth? 😅

Reply:
nope, we do not :D